### ISSUE TITLE
have no Forecasting example?

### ISSUE THREAD

**Issue by xinsuinizhuan** [Artelnics/opennn#211]
before airline_passengers and temperature_forecasting is avalibale, but now i can't find them

**Comment by xinsuinizhuan** [comment]
forecasting could provide some examples?

**Comment by davidge807** [comment]
Hi @xinsuinizhuan 

We have just added "Airline Passengers Example" to branch "dev". You can find it in the commit number `c97828db568d5b2b8ab74389bac7ebcd1d1d35c0`. We hope that it is useful for you.

Thank you very much for your comment and interest in OpenNN.

**Comment by xinsuinizhuan** [comment]
I have a time serial records:
one record: 97columns，first is timestamp of day, then 2-97 is record values, like:
![图片](https://user-images.githubusercontent.com/40679769/158132346-3cfa5308-2297-4b12-8f49-4c2d305bdf4b.png)
I want use the pre 10 records, to predict the eleventh record, use the Forecasting, but the parameters don't know what mean and how to set, the code have no annotation, how should i do to set the lags_number and steps_ahead_number,as the airline_passengers example
![图片](https://user-images.githubusercontent.com/40679769/158132769-7aed9adb-612e-4304-8aaa-01c7b678a5fc.png)

**Comment by xinsuinizhuan** [comment]
need set the first one colums the DateTime type?as:
data_set.set_column_type(0, DataSet::ColumnType::DateTime)?
And how to use lstm, the forecasting default has the lstm?

**Comment by davidge807** [comment]
There's no need to set the first column as DateTime type. Furthermore, your column doesn't adjust to a DateTime format (YYYY/MM/DD)

Instead, we recommend you to set it as "Unused" by: `data_set.set_column_set(0, DataSet::VariableUse::Unused)`.

By default, the Forecasting project type includes an LSTM layer, followed by a perceptron one.

Thank you for your comment and interest in the OpenNN library.

**Comment by xinsuinizhuan** [comment]
But i think the first colum daystamp is import, it have the import relatedness  with the last 96 records value. Others, could i add the lstm layer and dropout layer by myslef?

**Comment by xinsuinizhuan** [comment]
I think the fisrt colum is a import element，how to use it? I could convert it to DateTime format (YYYY/MM/DD)

**Comment by xinsuinizhuan** [comment]
I want to add to three  LSTM layers and perceptrons,like:
# build the model
model = Sequential()
# layer 1: LSTM
model.add(LSTM( input_dim=1, output_dim=50, return_sequences=True))
model.add(Dropout(0.2))
# layer 2: LSTM
model.add(LSTM(output_dim=100, return_sequences=False))
model.add(Dropout(0.2))
# layer 3: dense
# linear activation: a(x) = x
model.add(Dense(output_dim=1, activation='linear'))
# compile the model
model.compile(loss="mse", optimizer="rmsprop")

**Comment by davidge807** [comment]
Hi @xinsuinizhuan 

### Datetime column 
If you want to use this column as an input, you can leave it as default. OpenNN will use it as a forecasting input.

        DataSet data_set("../data/airline_passengers.csv", ',', true);`

        const Index lags_number = 2;
        const Index steps_ahead_number = 1;

        data_set.set_lags_number(lags_number);
        data_set.set_steps_ahead_number(steps_ahead_number);
        data_set.transform_time_series();

### Model building
Unfortunately, the dropout layer is not available in the OpenNN library. However, you can use the following code for creating the model:

        NeuralNetwork neural_network;
        
        LongShortTermMemoryLayer* lstm_layer_1(inputs_number, neurons_number_1);
        LongShortTermMemoryLayer* lstm_layer_2(neurons_number_1, neurons_number_2);
        PerceptronLayer* perceptron_layer(neurons_number_2, neurons_number_3);
        
        neural_network.add_layer(lstm_layer_1);
        neural_network.add_layer(lstm_layer_2);
        neural_network.add_layer(perceptron_layer);
        
        TrainingStrategy training_strategy(&neural_network, &data_set);
        training_strategy.set_loss_method(TrainingStrategy::LossMethod::MEAN_SQUARED_ERROR);
        training_strategy.set_optimization_method(TrainingStrategy::OptimizationMethod::QUASI_NEWTON_METHOD);
        training_strategy.perform_training();
